README.md

```diff
@@ -12,7 +12,71 @@ This module is used by the CI tests of the different Five Bells components. This
 npm install --save-dev five-bells-integration-test
 ```
 
-## Usage
+## Usage (with Docker)
+Get the Docker image for the Five Bells integration tests,
```

Comment:
I think you should include a warning that this will pull about 10 gigabytes of data